include/swift/Frontend/FrontendInputsAndOutputs.h

```diff
@@ -79,6 +84,9 @@ class FrontendInputsAndOutputs {
 
   bool isWholeModule() const { return !hasPrimaryInputs(); }
 
+  // FIXME: dmu fix uses / remove these when batch mode works
```

Comment:
Nitpick: please don't leave the personal FIXMEs in the code. If you want to leave a more long-lasting, complete-sentence one, that's fine.